test/s3select_test.cpp

```diff
@@ -2827,3 +2915,32 @@ phoneNumbers.addr. : 88
 
 }
 
+TEST(TestS3selectFunctions, csv_json_count)
+{
+  std::string input;
+  std::string result;
+  size_t size = 5;
+  generate_csv(input, size);
+  std::string js = convert_to_json(input.c_str(), input.size());
+  std::string input_query_1 = "select count(0) from s3object;";
+  const char* json_query_1 = convert_query(input_query_1);
+
+  run_json_query(json_query_1, js, result);
+  ASSERT_EQ(result, "5");
```

Comment:
it needs to compare that CSV result equal to JSON result just as in Parquet (Parquet = JSON)